auth/README.md

```diff
@@ -146,6 +149,9 @@ SMQ_JAEGER_URL=http://localhost:14268/api/traces \
 SMQ_JAEGER_TRACE_RATIO=1.0 \
 SMQ_SEND_TELEMETRY=true \
 SMQ_AUTH_ADAPTER_INSTANCE_ID="" \
+SMQ_AUTH_CALLOUT_URLS="" \
+SMQ_AUTH_CALLOUT_METHOD="POST" \
+SMQ_AUTH_CALLOUT_TLS_VERIFICATION=true \
```

Comment:
What do you think about having option for request timeout ?

Reply:
Done